infra/nginx/drc.conf

```diff
@@ -18,11 +18,6 @@ server {
     }
 
     error_page  404              /404.html;
-
-    # redirect server error pages to the static page /50x.html
-    #
-    error_page   500 502 503 504  /50x.html;
-    location = /50x.html {
-        root   /usr/share/nginx/html;
-    }
+    error_page  413              /413.json;
+    error_page   500 502 503 504  /500.json;
```

Comment:
Graag limiteren tot `location /api { ... }` en voor de rest html error pagina's?

Reply:
Overeengekomen dat de configuratie-overhead het niet waard is.